compiler/rustc_codegen_gcc/build_system/src/config.rs

```diff
@@ -5,7 +5,6 @@ use std::env as std_env;
 pub struct ConfigInfo {
     pub target: String,
     pub target_triple: String,
-    pub rustc_command: Vec<String>,
```

Comment:
This might not be used in this repo for now, but it is used in the [subtree](https://github.com/rust-lang/rustc_codegen_gcc), so this will be used in the next sync, so I believe it is better to keep it.